src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -60,6 +60,9 @@
     public static final int MAX_RECORDS_IN_RAM = 500000;
 
     private static final char[] RESERVED_ATTRIBUTE_STARTS = {'X', 'Y', 'Z'};
+    static final String HARD_CLIPPED_BASES_TAG = "XB";
```

Comment:
perhaps let's use non-X (but lowercase) tag, so that we can try to make it a real (upper-cased) tag in hts-specs later. look for a reasonable two-caracter tag name...perhaps cb and cq is not taken? also, need to think what to do if it's clipped from both ends....